### DISCUSSION TITLE
staticPlugin response NOT_FOUND

### DISCUSSION THREAD

**Discussion by goodlythink** [elysiajs/elysia#735] [Q&A]
I have tried following the example.

```
import { Elysia } from 'elysia'
import { staticPlugin } from '@elysiajs/static'

new Elysia()
    .use(staticPlugin())
    .get("/", () => "Hello World!")
    .listen(3000)
```
    
and my folder structure
- public
   - student.png
- src
  - index.ts
  
 when run on browser
**http://localhost:3000/public/student.png**

browser display **NOT_FOUND**

**how to fix**

![image](https://github.com/user-attachments/assets/a9daba67-c6f9-487a-b7fc-7f02230ff57c)

![image](https://github.com/user-attachments/assets/50c657c7-0568-4f2d-bc15-849f2cb19d7e)

**Comment by kravetsone** [comment]
why not issue?

**Comment by goodlythink** [comment]
Ok', i'm create new issue
